cancer_ai/validator/competition_manager.py

```diff
@@ -148,18 +155,20 @@ async def chain_miner_to_model_info(
         )
         return model_info
 
-    async def sync_chain_miners_test(self):
+    async def get_mock_miner_models(self):
         """Get registered mineres from testnet subnet 163"""
         self.model_manager.hotkey_store = get_mock_hotkeys_with_models()
 
-    async def sync_chain_miners(self):
+    async def get_miner_models(self):
```

Comment:
I think previous name was better because the function doesnt returns anything, it updates some state, so sync/update would be more descriptive

Reply:
they are not taken from chain, they are already in the database. I changed it to "update_miner_models"